redbot/cogs/image/image.py

```diff
@@ -27,6 +27,13 @@ def __init__(self, bot):
     def __unload(self):
         self.session.detach()
 
+    async def initialize(self) -> None:
+        """Move the API keys from cog stored config to core bot config if they exist."""
+        imgur_token = await self.settings.imgur_client_id()
+        if imgur_token is not None and "imgur" not in await self.bot.db.api_tokens():
+            await self.bot.db.api_tokens.set_raw("imgur", value={"client_id": imgur_token})
+            await self.config.settings.imgur_client_id.set(None)
```

Comment:
Rather than setting the key to `None`, to reset a key to its default, it's best to completely remove the key from storage with `Value.clear()`.
```suggestion
            await self.config.settings.imgur_client_id.clear()
```